nginx/self-signed-cert.sls

```diff
@@ -0,0 +1,25 @@
+self_signed_cert:
```

Comment:
This can be refactored into https://github.com/jcu-eresearch/shared-salt-states/blob/master/nginx/config.sls - which is pillar-based config for different hosts.  Note that each of the state's names need to be unique so they'll be prefixed with `{{ host }}`.

Reply:
This change has been made and the entire `self-signed-cert` file has been removed.